doc/source/advanced/datatype.rst

```diff
@@ -0,0 +1,90 @@
+***********************************
+Data Types in Lux
```

Comment:
`Defining Data Type`, no need `in Lux` since it's clear what we're referring to here.

Reply:
Ahh I wanted to specify in Lux so that its clear from the get-go that this page is about Lux Data Types and not dtype